google-datacatalog-rdbms-connector/src/google/datacatalog_connectors/rdbms/scrape/base_metadata_enricher.py

```diff
@@ -16,8 +16,9 @@
 
 import re
 
+from google.datacatalog_connectors.rdbms.common import constants
 from google.datacatalog_connectors.rdbms.scrape import \
```

Comment:
```suggestion
from google.datacatalog_connectors.rdbms import scrape
```

Reply:
I didn't change this one, because doing so generates a cyclic import, since the file is inside the scrape package.